server/http_handler.go

```diff
@@ -667,6 +671,81 @@ func (h binlogRecover) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	binloginfo.DisableSkipBinlogFlag()
 }
 
+type tableFlashReplicaInfo struct {
+	ID             int64    `json:"id"`
+	ReplicaCount   uint64   `json:"replica_count"`
+	LocationLabels []string `json:"location_labels"`
+	Status         bool     `json:"status"`
```

Comment:
IMHO it's better to set rename this variable to `Available`:
`Available := true` / `false`
or change its type to `string`:
`Status := "available"` / `"not_available"`